src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -1519,4 +1519,25 @@ Click CANCEL to cancel the update.</value>
   <data name="TooltipCurrentIndex" xml:space="preserve">
     <value>{0} of {1}</value>
   </data>
+  <data name="DSFunctionNodeDescription" xml:space="preserve">
+    <value>DesignScript Builtin Functions</value>
+  </data>
+  <data name="DSVarArgFuncBuiltin" xml:space="preserve">
+    <value>DesignScript Builtin Functions</value>
+  </data>
+  <data name="CustomNodeDescription" xml:space="preserve">
+    <value>Instance of a Custom Node</value>
+  </data>
+  <data name="Point2Node" xml:space="preserve">
+    <value>Create a point in the XY plane.</value>
+  </data>
+  <data name="Point3Node" xml:space="preserve">
+    <value>Create a point in 3D space.</value>
+  </data>
+  <data name="OutputNodeDescription" xml:space="preserve">
+    <value>A function output, use with custom nodes</value>
+  </data>
+  <data name="SymbolNodeDescription" xml:space="preserve">
+    <value>A function parameter, use with custom nodes</value>
+  </data>
```

Comment:
`SymbolNodeDescription` being used twice, but there are differences in the original string contents, do use the longer version.